src/software/ai/evaluation/deflect_off_enemy_target.cpp

```diff
@@ -39,7 +39,10 @@ namespace Evaluation
             {
                 if (fabs(i.position().y() - closestEdgeY) < shortestLenToEdge)
                 {
-                    enemyClosestToEdge.updateState(i);
+                    enemyClosestToEdge.updateCurrentState(
```

Comment:
Instead of constructing a new ```RobotState``` with the Robot attributes, might be better to use ```i.currentState()``` ?

Reply:
Hmm, seems like we would want the enemy history here as well, so why not `enemyClosestToEdge = i;`?

Reply:
done